src/Raven.Client/Documents/Conventions/DocumentConventions.cs

```diff
@@ -44,7 +44,7 @@ public class DocumentConventions : Client.Conventions
             SendApplicationIdentifier = false,
             MaxContextSizeToKeep = new Size(PlatformDetails.Is32Bits == false ? 8 : 2, SizeUnit.Megabytes),
 #if NETCOREAPP3_1_OR_GREATER
-            HttpPooledConnectionLifetime = TimeSpan.FromMinutes(3)
+            HttpPooledConnectionLifetime = TimeSpan.FromMinutes(19)
```

Comment:
`HttpPooledConnectionLifetime = TimeSpan.FromMinutes(19) + TimeSpan.FromSeconds(39) + TimeSpan.FromMilliseconds(9)`

Not really, but funny